estimate.src.ipynb

```diff
@@ -729,9 +720,8 @@
    "outputs": [],
    "source": [
     "fig, ax = plt.subplots(figsize=figsize)\n",
-    "for i, (name, column) in enumerate(residual_df.iteritems()):\n",
-    "    ax.scatter(residual_df.index, column, c=mpl.rcParams['axes.color_cycle'][i], label=name)\n",
-    "ax.legend(loc='upper left')\n",
+    "ret = ax.plot(residual_df, 'o', ms=2)\n",
```

Comment:
turns out that you can just pass a dataframe to ax.plot() and it magically unpacks stuff and formats the axes to be datetimes. @tacaswell is my hero